### PR TITLE
Use box-shadow for sticky table header

### PR DESCRIPTION
**Pull request by BriungRi**
Possible improvements:

1. Use tailwind -- done
2. Access color variable instead of hardcode -- done

### CODE REVIEW

File: src/typescript/frontend/src/scratch.ts.bak
Comment (chiumax) [inline src/typescript/frontend/src/scratch.ts.bak:1]: this file here on purpose?

**Comment by chiumax** [comment]
uses tailwind: https://tailwindcss.com/docs/box-shadow#arbitrary-values

**Review by chiumax — APPROVED** [review]
anyways, i like this

**Comment by bogberry** [comment]
I think this is better, nice work!

**Pull request merged**